iree/turbine/kernel/wave/index_sequence_analysis.py

```diff
@@ -219,6 +219,9 @@ def set_vector_shapes(
             or node in mma_slices[mma][MMA_LHS]
             or node in mma_slices[mma][MMA_RHS]
         ):
+            # Ensure that the operators indexing dims are present in the anchor.
```

Comment:
Can you remind me what happen if we don't observe this again?

Reply:
sure will update the comment.